web/vtadmin/package.json

```diff
@@ -53,7 +54,9 @@
     "lint:stylelint": "stylelint \"**/*.{css,scss}\"",
     "lint:stylelint:fix": "npm run lint:stylelint -- --fix",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "build:planetscale-vtadmin": "tsc -p ./planetscale-vtadmin-template/tsconfig.json --outDir ./planetscale-vtadmin && cp ./planetscale-vtadmin-template/README.md ./planetscale-vtadmin && node ./scripts/verify-pscale-build.ts",
```

Comment:
These scripts are only ever used by planetscale to release @planetscale/vtadmin and should not cause merge conflicts.